src/components/targets-map/TargetsMap.js

```diff
@@ -53,4 +54,10 @@ const TargetsMap = ({
   );
 };
 
+TargetsMap.propTypes = {
+  newTargetlatlng: PropTypes.object,
```

Comment:
for these cases you can follow tami's advice and define the shape of your object, I don't remember exactly the shape of the newTargetlatlng, but It would be something like:
TargetsMap.propTypes = {
  newTargetlatlng: shape({
  lat: number,
  lng: number,
  ...
})
}